helm/kagent/values.yaml

```diff
@@ -46,6 +46,9 @@ tolerations: []
 # -- Node labels to match for `Pod` [scheduling](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/).
 nodeSelector: {}
 
+# Affinities for `Pod` https://kubernetes.io/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity/
+affinity: {}
```

Comment:
Are these actually used anywhere? The templates seem to just check their namespaces values